packages/design-system/src/molecules/walletCard/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { StyleSheet, View } from 'react-native'
+import { useMemo } from 'react'
+import { useThemeV2 } from '../../hooks'
+import { container, margin, spacing } from '../../styles'
+import { CircleImg, Icon, TypographyV2 } from '../../atoms'
+
+export interface WalletCardProps {
+  tokenSymbol: string
+  tokenName: string
+  tokenLogo: string
+  balance: string
+  secondaryBalance: string
+  onPress?: () => void
```

Comment:
I think you forgot to implement the `onPress` inside the component

Reply:
✅